### DISCUSSION TITLE
v1.123.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#14748] [Announcements]
# v1.123.0

## Highlights

Welcome to release `v1.123.0` of Immich. This is the last release of the year as the team is winding down for the holiday season. Thank you for your support and the fantastic year. We cannot wait to continue adding more features, optimizing, and making Immich a delightful app experience for you and your family. 

This release fixes bugs in the web and mobile, notably from the recent new native video player. We also add a feature to allow you to change the timeline view order of an album in the mobile app

<p align="center">
<img src="https://github.com/user-attachments/assets/7e47dc58-cfeb-4635-8dba-ca7770abef04"  width="300"/>
</p>

## Support Immich

<p align="center">

<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExbjY2eWc5Y2F0ZW56MmR4aWE0dDhzZXlidXRmYWZyajl1bWZidXZpcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/87CKDqErVfMqY/giphy.gif" width="450" title="SUPPORT THE PROJECT!"> 

</p>

If you find the project helpful, you can support Immich by purchasing a product key at https://buy.immich.app/.

Cheers! 🍻

- - - -

### 🚀 Features
* feat(server): Merge Faces sorted by Similarity by @Lukasdotcom in https://github.com/immich-app/immich/pull/14635
* feat(album): album view sort order by @alextran1502 in https://github.com/immich-app/immich/pull/14648
### 🌟 Enhancements
* feat(server): add Bash healthcheck script by @000yesnt in https://github.com/immich-app/immich/pull/14704
### 🐛 Bug fixes
* fix(web): live photo link action by @michelheusschen in https://github.com/immich-app/immich/pull/14668
* fix(web): allow minimizing upload panel by @michelheusschen in https://github.com/immich-app/immich/pull/14663
* fix(web): upload info panel covers timeline navigation bar by @alextran1502 in https://github.com/immich-app/immich/pull/14651
* fix(server): fixed email footer image stretched #14617 by @TimVanOnckelen in https://github.com/immich-app/immich/pull/14671
* fix(mobile): set custom headers on external url (#14707) by @alex-sherwin in https://github.com/immich-app/immich/pull/14708
* fix(mobile): native dialog location by @alextran1502 in https://github.com/immich-app/immich/pull/14730
* fix(mobile): live photos playback by @alextran1502 in https://github.com/immich-app/immich/pull/14731
* fix(web): fix missing icons  by @aviv926 in https://github.com/immich-app/immich/pull/14737
### 📚 Documentation
* docs: improve install/requirements and install/docker-compose by @pyorot in https://github.com/immich-app/immich/pull/13569
* docs: Add more documentation for video transcoding settings. by @kevincox in https://github.com/immich-app/immich/pull/13388
### 🌐 Translations
* chore(web): update translations by @weblate in https://github.com/immich-app/immich/pull/14495

## New Contributors
* @alex-sherwin made their first contribution in https://github.com/immich-app/immich/pull/14708
* @000yesnt made their first contribution in https://github.com/immich-app/immich/pull/14704

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.122.3...v1.123.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.123.0'>v1.123.0</a>.</em>

**Comment by HELPY4** [comment]
Happy New Year! :)

**Comment by Dawo9889** [comment]
Merry Christmas!

**Comment by BigPebble123** [comment]
You guys are doing a great job! Enjoy your holidays :smile:

**Comment by zzzdaxia** [comment]
Have a nice holiday~！

**Comment by MMK21Hub** [comment]
Thanks for adding the Bash healthcheck script! It reduced the average CPU usage (during "idle" times) from 14.8% to 10.8% on my Raspberry Pi 400.

![Immich updated CPU usage v1 123 0](https://github.com/user-attachments/assets/70f524a3-bca3-44c4-b79b-6f8fea6ea81a)

**Comment by Zhang-abab** [comment]
You guys are doing a great job! Enjoy your holidays

**Comment by MikNolhan** [comment]
Anyone else notice issues with the mobile app? Constant crashes and black screen. Freezes at every menu. Don't want to clear the cache if I don't have to given the size of my library, but the app is basically non-functional since I updated, and eating up power.


Cleared cache. Briefly worked. Then stops working again. [![Screenshot_20241218-143034.png](https://github.com/user-attachments/assets/3d27854e-a7be-4e81-9c51-8c3dd3d824b2)

Worked fine prior to the update. Haven't had this issue before with other updates.

**Reply by bo0tzz** [reply]
Are you on app version v1.123.0 exactly?

**Comment by MikNolhan** [comment]
Yeah. It definitely is a cache issue. Waiting overnight helped. I even removed the app and rebooted and reinstalled.

It hasn't even loaded any albums though. Super annoying. I can open a bug report.

Dec 19, 2024 03:56:29 bo0tzz ***@***.***>:

**Comment by welpher** [comment]
Happy New Year! :)

> Merge Faces sorted by Similarity。

On the merge page, I just merge an unnamed face into a named face not into a similar face. And affected the display order of faces in photo detail page . Is there a setting somewhere to change the display order of faces in photo detain page and the merge page?
